.github/pull_request_template.md

```diff
@@ -1,16 +1,11 @@
-**Which issue is resolved by this Pull Request:**
-Resolves #
-
-<!-- You can delete this section if no corresponding issue -->
-
 **Description of your changes:**
 
 
 **Checklist:**
-- Do you want this PR cherry picked to release branch?
+- [ ] Do you want this PR cherry picked to release branch?
 
     If yes, please either
-    * ask the approver to add label `cherrypick-approved`, so that release manager
+    * (recommended) ask the approver to add label `cherrypick-approved`, so that release manager
```

Comment:
Suggested revision:

If yes, use one of the following options:
  
* **(Recommended.)** Ask the PR approver to add the `cherrypick-approved` label to this PR. The release manager adds this PR to the release branch in a batch update.
*  After this PR is merged, create a cherry-pick PR to add these changes to the release branch. (For more information about creating a cherry-pick PR, see the [Kubeflow Pipelines release guide](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md).)